chat.go

```diff
@@ -272,6 +335,25 @@ func MsgOptionDisableMarkdown() MsgOption {
 	}
 }
 
+func MsgOptionPostEphemeralParameters(params PostEphemeralParameters) MsgOption {
```

Comment:
Why do we need both this and `MsgOptionPostParameters`? Do ephemeral messages not support the options handled there?

Reply:
Technically we don't, we could probably use the standard `MsgOptionsPostMessageParameters` function but in case this deviated in future it would be good to have its own space.

Reply:
I'd prefer we re-use code as much as possible (less code, less bugs etc) and the day the APIs deviate we'll refactor.